### PR TITLE
[CMS-791] Copy pantheon-mu-plugin from our repo

### PR DESCRIPTION
**Pull request by jazzsequence**
### Overview
This pull request:
Updates the `wp` update step to actually run the filters (previously filters were included in the workflow but never called in the update method), and adds a new filter that pulls down the latest version of [pantheon-systems/pantheon-mu-plugin](https://github.com/pantheon-systems/pantheon-mu-plugin) as part of the wp update.

This PR also updates the `CHANGELOG.md` to just point to the Releases page, since the CHANGELOG was not being maintained.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Has tests?    | yes
| BC breaks?    | no     
| Deprecations? | no

### CODE REVIEW

File: src/Update/Filters/CopyMuPlugin.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        if ($fs->exists($muplugin_dir . '/pantheon')) {
+            $this->logger->notice('Removing {file}', ['file' => $dest . '/pantheon']);
+            $fs->remove($dest . '/pantheon');
+        }
+    }
+
+    /**
+     * Helpful wrapper to call either 'mirror' or 'copy' as needed.
+     */
+    protected function copyFileOrDirectory($src, $dest)
```
Comment (kporras07) [inline src/Update/Filters/CopyMuPlugin.php:76]: This is not used, is it?

**Review by kporras07 — COMMENTED** [review]
Looks pretty good, just a couple of comments

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,3 @@
...
-### Unreleased
-
-* Remove rpmbuild-php (#45)
-* Add body text to the PR for automated cos-php updates (#47)
-* Automate core version update in WP Launch Check (#64)
-
-### 0.1.0 - 2018/May/26
-
-* Created from template `pantheon-systems/updatinate`
+See [Releases](https://github.com/pantheon-systems/releases) for the latest release notes.
```
[the remark this answers is not in the document]
Comment (jazzsequence) [reply CHANGELOG.md:3]: 🤦‍♂️

File: src/Update/Filters/CopyMuPlugin.php [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            "$dest/.git",
+            "$dest/README.md",
+            "$dest/composer.json",
+        ];
+        foreach ($files_to_delete as $file) {
+            $this->logger->notice('Removing {file}', ['file' => $file]);
+            $fs->remove($file);
+        }
+
+        // Check if the /pantheon subdirecotry exists. If it does, delete that, too.
```
[the remark this answers is not in the document]
Comment (jazzsequence) [reply src/Update/Filters/CopyMuPlugin.php:66]:
```suggestion
        // Check if the /pantheon subdirectory exists. If it does, delete that, too.
```

File: src/Update/Filters/CopyMuPlugin.php (hunk 1 above)
Comment (jazzsequence) [reply src/Update/Filters/CopyMuPlugin.php:76]: Correct, it was left behind from making a copy of CopyPlatformAdditions.php and then I never went back to delete it. 👍

**Review by kporras07 — APPROVED** [review]
LGTM

**Pull request merged**